digicampipe/utils/geometry.py

```diff
@@ -1,9 +1,8 @@
 from cts_core import camera as cam
 import numpy as np
 import astropy.units as u
-from ctapipe.instrument.camera import CameraGeometry
-from ctapipe.instrument.camera import _find_neighbor_pixels as find_neighbor_pixels
-
+from digicampipe.instrument.camera import CameraGeometry
```

Comment:
I just see this was a mistake on my side... should not haven ended up in this PR. 
Will remove it soon ... at the moment I am traveling and only have a phone, so in order not to forget this ... just a comment.